functable.c

```diff
@@ -12,14 +12,14 @@
 
 /* insert_string */
 #ifdef X86_SSE4_2_CRC_HASH
-extern Pos insert_string_sse(deflate_state *const s, const Pos str, unsigned int count);
+ZNG_TARGET_SSE42 extern Pos insert_string_sse(deflate_state *const s, const Pos str, unsigned int count);
```

Comment:
IIRC, the target attribute is not needed for the extern definitions, as it does not change the function definition itself.

Reply:
Good catch. Thanks.